modules/custom/az_migration/config/install/migrate_plus.migration.az_node_person.yml

```diff
@@ -0,0 +1,150 @@
+id: az_node_person
+label: Person Node Content
+migration_group: az_migration
+status: true
+
+source:
+  plugin: az_person
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_person
+  migration_dependencies:
+    requireed:
+      - az_person_categories
+      - az_person_categories_secondary
+
+process:
+  nid: nid
+  vid: vid
+  type: 
+    plugin: default_value
+    default_value: az_person
+  langcode:
+    plugin: static_map
+    bypass: true
+    source: language
+    map:
+      und: en
+      
+  'field_az_fname/value': person_fname_value
+  'field_az_fname/format': person_fname_format
+
+  'field_az_lname/value': person_lname_value
+  'field_az_lname/format': person_lname_format
+
+  'field_az_email/value' : person_email
+
+  field_az_titles:
+    plugin: sub_process
+    source: person_job_title
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_degrees:
+    plugin: sub_process
+    source: person_degrees
+    process:
+      delta: delta
+      value: value
+      format: format
+
+  field_az_phones:
+    plugin: sub_process
+    source: person_phones
+    process:
+      delta: delta
+      value: value
+
+  field_az_links:
+    plugin: sub_process
+    source: person_links
+    process:
+      delta: delta
+      uri: url
+      title: title
+
+  field_az_address:
+    plugin: sub_process
+    source: person_addresses
+    process:
+      delta: delta
+      value: value
+      summary: summary
+      format: 
+        plugin: default_value
+        default_value: az_standard
+
+  field_az_body:
+    plugin: sub_process
+    source: person_biography
+    process:
+      delta: delta
+      value: value
+      summary: summary
+      format:  
```

Comment:
We'll probably need another process for all fields that use az_standard to convert embedded entities here eventually.

Reply:
ok. for now, as per the discussion, tagged directly to "az_standard" format as default. Since the direct format mapping from d7 is not present.